### PR TITLE
Prepare to release v1.0.0-RC2 (stable) / v0.22.0 (unstable)

### PR DESCRIPTION
**Pull request by punya**
https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/releases/edit/untagged-51426c81a783f97c6b62

### CODE REVIEW

File: e2e-test-server/go.mod
Comment (dashpole) [inline e2e-test-server/go.mod:3]: Should we be downgrading the go version?

Comment (dashpole) [reply e2e-test-server/go.mod:3]: I see the commit message above.  Can you provide more details on what happened?

Comment (punya) [reply e2e-test-server/go.mod:3]: All our other go.mod files say 1.14. Because this one file says 1.16, my local Go (1.16) would always make a change in go.sum that would be rejected by our CI (which runs 1.14). I had to run `make precommit` under Go 1.14 using Docker to make CI happy.

Comment (dashpole) [reply e2e-test-server/go.mod:3]: Thanks.

**Review by dashpole — APPROVED** [review]

**Review by aabmass — APPROVED** [review]
Nice release notes!

**Comment by punya** [comment]
I just noticed that #200 is ready to merge, so I'll get that in and update this PR afterwards.

**Comment by ymotongpoo** [comment]
I think #200 requires another LGTM after reopening it.

**Comment by bogdandrutu** [comment]
@punya would be great to have this sooner than later :)

**Pull request merged**
